### PR TITLE
Add `RUNNER_GROUP` environment variable

### PR DESCRIPTION
**Pull request by ybiquitous**
The new variable will be used Runners. See https://github.com/sider/runners/pull/833

### CODE REVIEW

File: base/bin/prepare [hunk 1]
```diff
@@ -62,6 +62,11 @@ apt-get -qyy clean
 # Create a user
 adduser --system --shell /bin/bash --home "$RUNNER_USER_HOME" "$RUNNER_USER"
 
+# Verify the user and group
+id -u "$RUNNER_USER"
+user_and_group=$(groups "$RUNNER_USER")
```
Comment (ybiquitous) [inline base/bin/prepare]: [note] See also:

- [id(1)](https://linux.die.net/man/1/id)
- [groups(1)](https://linux.die.net/man/1/groups)

File: base/bin/prepare [hunk 2]
```diff
@@ -62,6 +62,11 @@ apt-get -qyy clean
...
+[[ "${user_and_group}" == "${RUNNER_USER} : ${RUNNER_GROUP}" ]] || (echo "${user_and_group}" ; exit 1)
```
Comment (yykamei) [inline base/bin/prepare]: In my laptop, groups(1) shows something like `docker kamei`; it does not look to contain `:`.
I'm curious about the output on GitHub Actions.

Comment (ybiquitous) [reply base/bin/prepare]: Oh, the output depends on platforms. Do you have any good ideas?

Comment (yykamei) [reply base/bin/prepare]: :thinking: What about getent(1)?

Example:

```console
$ getent group docker
docker:x:970:kamei
```

Comment (yykamei) [reply base/bin/prepare]: getent(1) depends on `/etc/group`, so I believe this command always shows the same format in any Linux distributions.

Comment (ybiquitous) [reply base/bin/prepare]: Sadly, the command prints nothing... 😢 

```
$ getent group analyzer_runner
```

But, `id(1)` seems available:

```
$ id --groups --name analyzer_runner
nogroup
```

What do you think about this idea?

Comment (yykamei) [reply base/bin/prepare]: I mean `getent group ${RUNNER_GROUP}`, but using id(1) with `--groups` and `--name` would work.

Comment (ybiquitous) [reply base/bin/prepare]: Sorry, I misunderstood... But `getent group ${RUNNER_GROUP}` doesn't seem to be able to verify if `$RUNNER_USER` belongs to `$RUNNER_GROUP`. 🤔 

So, I'll do it by `id --groups --name $RUNNER_USER`.

Comment (ybiquitous) [reply base/bin/prepare]: Thanks for your comment! I've fixed via 37d3c5c.

**Pull request merged**